src/state/slices/portfolioSlice/portfolioSlice.test.ts

```diff
@@ -255,44 +247,6 @@ describe('portfolioSlice', () => {
       })
     })
 
-    describe('selectAccountIdByAddress', () => {
```

Comment:
dead code - was going to remove in another PR but it's getting the treatment here